src/elements/partners-block.html

```diff
@@ -5,8 +5,6 @@
 <link rel="import" href="../../bower_components/iron-icon/iron-icon.html">
 
 <link rel="import" href="../redux-mixin.html">
-<link rel="import" href="./shared-styles.html">
```

Comment:
I think it's about reusable components. When the component should be independent, have everything inside it and ready to be used from any other places/projects etc.
But when we specify relative paths to other files in projects (not from bower_components folder), the component is dependent, so in that case we don't need to have such imports in every file, coz it gets that from parent